apps/contrib/templates/meinberlin_contrib/includes/map_filter_and_sort.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n contrib_tags %}
+
+<div class="filter-bar">
+    {% if mode == 'map' %}
+        {% combined_url_parameter request.GET mode='list' as url_par %}
+        <a class="button button--light" href="{{ url_par }}">
+            <i class="fa fa-map" aria-hidden="true"></i>
+            {% trans 'view' %}
+        </a>
+    {% else %}
+        {% combined_url_parameter request.GET mode='map' as url_par %}
+        <a class="button button--light" href="{{ url_par }}">
+            <i class="fa fa-list" aria-hidden="true"></i>
+            {% trans 'view' %}
```

Comment:
This is not accessible. We should overwrite the button's link text here with `aria-label="view as list"`